app/controllers/notifications_settings_controller.rb

```diff
@@ -15,7 +15,7 @@ def unsubscribe_from_emails
   def subscribe_to_emails
     matching_intakes = matching_intakes(params[:email_address])
 
-    if matching_intakes.present?
+    if matching_intakes.any?
```

Comment:
Probably minor, but I think if we aren't doing `#to_a` to force an array correction, you can avoid looping and do a bulk update. Maybe not even need the check at all and allow the operation to proceed on an empty collection, which might work. Not sure off the top of my head.